logistics_project/apps/malawi/migrations/0015_auto__add_field_reportingrate_is_facility.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'ReportingRate.is_facility'
+        db.add_column('malawi_reportingrate', 'is_facility',
+                      self.gf('django.db.models.fields.BooleanField')(default=False),
+                      keep_default=False)
```

Comment:
we can discuss offline but we should check to see how many records are on this table and maybe run the migration separate from deploy if needed